docs/site/src/plugins/inject-code/injectLoader.js

```diff
@@ -54,10 +54,16 @@ const addCodeInject = function (source) {
             case "rs":
               language = "rust";
               break;
+            case "prisma":
+              language = "ts";
+              break;
             default:
               language = fileExt;
           }
 
+          const isMove = language === "move" ? true : false;
+          const isTs = language === "ts" || language === "js" ? true : false;
```

Comment:
Should be able to drop the ternaries here.

```suggestion
          const isMove = language === "move";
          const isTs = language === "ts" || language === "js";
```